paper/paper.md

```diff
@@ -1,124 +1,240 @@
-Efficiency and Renewable Energy Postdoctoral Fellowship Program
-(2014-2016), Tucson Electric Power, Arizona Public Service, and Public
-Service Company of New Mexico (2016-2018), and University of Arizona
-Institute for Energy Solutions (2017-2018).
+pvlib python is a community-developed, open-source software toolbox
+for simulating the performance of solar photovoltaic (PV) energy
+components and systems.  It provides reference implementations of
+over 100 empirical and physics-based models from the peer-reviewed scientific literature,
+including solar position algorithms, irradiance models, thermal models,
+and PV electrical models.  In addition to individual low-level
+model implementations, pvlib python provides high-level workflows
+that chain these models together like building blocks to form complete
+"weather-to-power" photovoltaic system models.  It also provides functions to
+fetch and import a wide variety of weather datasets useful for PV modeling. 
+
+pvlib python has been developed since 2013 and follows modern best
+practices for open-source python software, with comprehensive automated
+testing, standards-based packaging, and semantic versioning.
+Its source code is developed openly on GitHub and releases are
+distributed via the Python Package Index (PyPI) and the conda-forge
+repository.  pvlib python's source code is made freely available under
+the permissive BSD-3 license.
+
+Here we (the project's core developers) present an update on pvlib python,
+describing capability and community development since our 2018 publication
+[@pvlibjoss2018].
+
+
+# Statement of need
+
+PV performance models are used throughout the field of photovoltaics.
+The rapid increase in scale, technological diversity, and sophistication
+of the global solar energy industry demands correspondingly more
+capable models.  Per the United States Department of Energy,
+"the importance of accurate modeling is hard to overstate" [@seto2022].
+
+Compared with other PV modeling tools, pvlib python stands out in several
+key aspects.  One is its toolbox design, providing the user a
+level of flexibility and customization beyond that of other tools.  Rather than organizing
+the user interface around pre-built modeling workflows, pvlib python
+makes the individual "building blocks" of PV performance models accessible to
+the user.  This allows the user to assemble their own model workflows, including
+the ability of incorporating custom modeling steps.  This flexibility
+is essential for applications in both academia and industry.
+
+Another key aspect of pvlib python is that it is used via
+a general-purpose programming language (Python), which
+allows pvlib python functions to be combined with capabilities in other Python packages,
+such as database query, data manipulation, numerical optimization,
+plotting, and reporting packages.
+
+A final key aspect of pvlib python is its open peer review approach and
```

Comment:
Maybe say, “foundation *on* published…” versus “in” because I think of building on top of foundations?